keycloak/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 name: keycloak
-version: 4.12.0
```

Comment:
Deprecating this chart is kind of a breaking change. I'd prefer to update the major version.